adagrams/game.py

```diff
@@ -1,11 +1,134 @@
+import random
+SCORE_CHART = {
+        'A': 1, 
+        'B': 3, 
+        'C': 3, 
+        'D': 2, 
+        'E': 1, 
+        'F': 4, 
+        'G': 2, 
+        'H': 4, 
+        'I': 1, 
+        'J': 8, 
+        'K': 5, 
+        'L': 1, 
+        'M': 3, 
+        'N': 1, 
+        'O': 1, 
+        'P': 3, 
+        'Q': 10, 
+        'R': 1, 
+        'S': 1, 
+        'T': 1, 
+        'U': 1, 
+        'V': 4, 
+        'W': 4, 
+        'X': 8, 
+        'Y': 4, 
+        'Z': 10
+    }
+
 def draw_letters():
-    pass
+    """ Draw letter from letter_pool """
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    letters = []
+   
+   # Draw a random letter
+    while len(letters) < 10:
+        letter = random.choice(list(LETTER_POOL))
+        if LETTER_POOL[letter] == 0: # Check letter is in pool
+            continue
+        else: # Remove one letter from the letter pool
+            letters.append(letter)
+            LETTER_POOL[letter] -= 1
+   
+    return letters
```

Comment:
One thing to consider: the way this function is structured, this will prevent us from picking a letter that is not available. However, when we randomly pick on L65, it's a random pick from the 26 letters in an alphabet. That means `A` and `Z`  both have 1/26th chance of being picked before we do the validation check, despite the letter bag distribution for `A` being 9/98 and `Z` being 1/98.